pinecone/index_connection_test.go

```diff
@@ -90,7 +90,6 @@ func (ts *IntegrationTests) TestDeleteVectorsByFilter() {
 
 	if ts.indexType == "serverless" {
 		assert.Error(ts.T(), err)
-		assert.Containsf(ts.T(), err.Error(), "Serverless and Starter indexes do not support deleting with metadata filtering", "Expected error message to contain 'Serverless and Starter indexes do not support deleting with metadata filtering'")
```

Comment:
This error message has changed slightly since this test was originally written. I think it's fine to remove the string check for now and just validate that we've gotten an error back.